Tests/SwiftSyntaxMacroExpansionTest/MemberMacroTests.swift

```diff
@@ -299,4 +299,62 @@ final class MemberMacroTests: XCTestCase {
       ]
     )
   }
+
+  func testAddMemberToEmptyDeclaration() {
+    struct TestMacro: MemberMacro {
+      static func expansion(
+        of node: AttributeSyntax,
+        providingMembersOf declaration: some DeclGroupSyntax,
+        conformingTo protocols: [TypeSyntax],
+        in context: some MacroExpansionContext
+      ) throws -> [DeclSyntax] {
+        return [DeclSyntax("var x = 0")]
+      }
+    }
+
+    assertMacroExpansion(
+      """
+      @Test
+      struct Foo {}
+      """,
+      expandedSource: """
+        struct Foo {
+
+          var x = 0}
+        """,
```

Comment:
My expectation is that this should be:
```
struct Foo {
  var x = 0
}
```

Same in the next test as well. Am I missing something?

Reply:
Got it now.